android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.pauldemarco.flutter_blue">
-  <uses-sdk android:minSdkVersion="18" />
+  <uses-sdk android:minSdkVersion="19" />
   <uses-permission android:name="android.permission.BLUETOOTH" />
   <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
   <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
This permission need to be changed to ACCESS_FINE_LOCATION because targetSdkVersion 29 require fine location in order to be able to see bluetooth scan results